### PR TITLE
inline: add game result support

### PR DESCRIPTION
**Pull request by rapour**
[Game](https://core.telegram.org/bots/api#inlinequeryresultgame) is one of the possible types a bot can send as inline query result.

### CODE REVIEW

File: inline_types.go [hunk 1]
```diff
@@ -60,6 +60,14 @@ func (r *ResultBase) Process(b *Bot) {
 	}
 }
 
+// GameResult represents a game.
+type GameResult struct {
+	ResultBase
+
+	// Unique identifier of the game.
```
Comment (demget) [inline inline_types.go]: Documentation

File: inline_types.go [hunk 2]
```diff
@@ -60,6 +60,14 @@ func (r *ResultBase) Process(b *Bot) {
...
+	GameShortName string `json:"game_short_name"`
```
Comment (demget) [inline inline_types.go]: `ShortName`

**Review by demget — CHANGES_REQUESTED** [review]

File: inline_types.go (hunk 1 above)
Comment (rapour) [reply inline_types.go]: Done

File: inline_types.go (hunk 2 above)
Comment (rapour) [reply inline_types.go]: Done

**Pull request merged**